slippers/exceptions.py

```diff
@@ -0,0 +1,9 @@
+class SlippersError(Exception):
+    def __init__(self, msg: str, argument: str, component: str):
+        self.msg = msg
+        self.argument = argument
+        self.component = component
+        super().__init__(msg)
+
+    def __str__(self):
+        return f"{self.msg} (\"{self.component}.{self.argument}\")"
```

Comment:
```suggestion
        return f'{self.msg} ("{self.component}.{self.argument}")'
```
I would suggest this to avoid the backslashes.

Reply:
https://github.com/QuickRelease/slippers/pull/25/files#diff-666b0bc010f876e7ae1a9d243ee1ac2fa81cdcbc9db5a0cf2c732029489039bcR9